### ISSUE TITLE
🚧App crashes upon scene transition

### ISSUE THREAD

**Issue by kselena** [KinsonDigital/Velaptor#391]
### Complete The Item Below

- [X] I have updated the title without removing the 🚧 emoji.

### Description

After clicking the next scene button in the animated graphics scene, the program locks up and is not responding.   The animated graphics scene works fine.

**Steps to reproduce:**
1. Run Velaptor  testing app
2. Click next scene button until you reach the animated graphics scene
3. Observe that the animated graphics scene is working fine
4. Click the next scene button one more time
5. Observe that the application locks up.
<img width="752" alt="image" src="https://user-images.githubusercontent.com/96791049/196231390-7e8f7da2-f899-4d6b-b45d-b4c1c0a274e3.png">

**Expected results:**
1. Application doesn't crash and moves to the next scene with no issue.

### Acceptance Criteria

- [ ] App doesn't crash when it changes to any scene

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [X] Issue linked to the correct project _(if applicable)_.
- [X] Issue linked to the correct milestone _(if applicable)_.
- [ ] Draft pull request created and linked to this issue _(only required with code changes)_.

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                |
|---------------------|----------------------|
| Bug Fixes           | `🐛bug` |
| Breaking Changes    | `🧨breaking changes` |
| New Feature         | `✨new feature` |
| Workflow Changes    | `workflow` |
| Code Doc Changes    | `🗒️documentation/code` |
| Product Doc Changes | `📝documentation/product` |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label             |
|---------------------|-------------------|
| Low Priority        | `low priority`    |
| Medium Priority     | `medium priority` |
| High Priority       | `high priority`   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

**Comment by CalvinWilkinson** [comment]
Issue closed without any code changes.

**Reason:**
It was discovered that the application was crashing because the _**soft_oal.dll**_ library was not available for the application to play sound.  When moving to the sound scene, the scene was attempting to load the sound content.  Loading the sound content requires the use of the _**soft_oal.dll**_ library.  Since the library was not present, the application would was crashing.

The reason that the library was not present during QA testing is that when the application was published, the _**runtimes**_ folder that held the library was not being published with the application.  The published application was then added to the QA testing issue #390 which was used for QA testing.

Further publishing of the testing application will require that the _**runtimes**_ directory be manually added to the publish result and only after the entire solution has been built. The reason for the entire solution being built is that to produce the **_runtimes_** directory, the solution has to be built.
